examples/helm/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as helm from "@pulumi/kubernetes/helm";
+
+const nginx = new helm.v2.Chart("simple-nginx", {
+  // Represents chart `stable/nginx-lego@v0.3.1`.
+  repo: "stable", chart: "nginx-lego", version: "0.3.1",
+  values: {
+    // Override for the Chart's `values.yml` file. Zero out resource requests so it can be scheduled
+    // on our (wimpy) CI cluster.
+    nginx: {resources: null},
+    default: {resources: null},
+    lego: {resources: null},
+  },
+  transformations: [
+    // Don't expose any services publicly, i.e., turn all services into ClusterIP instead of
+    // LoadBalancer. (This cuts time to run CI tests significantly.)
+    helm.transform.ifGvkEqual("v1", "Service", (svc: any) => {
+      if (svc.spec && svc.spec.type && svc.spec.type == "LoadBalancer") {
+        svc.spec.type = "ClusterIP";
+      }
+    }),
+  ]
+});
```

Comment:
In general, I love it when these examples show off how you expose the externally addressable endpoint from the stack.  That helps complete the scenario ("deploy this then curl the endpoint it returns").  Can we do that here?  It also helps show off accessing outputs of constituent resources.

Reply:
When we move a similar example to `pulumi/examples`, yes, but we can't expose it publicly in CI until we stop using minikube, right?

Reply:
Possibly true - though since you are forcing to ClusterIP for now, can you provide a URL based on that?  (Not an export on how ClusterIP-based Service endpoints work...).